package.json

```diff
@@ -1229,7 +1229,7 @@
     "suricata-sid-db": "^1.0.2",
     "swr": "^2.2.5",
     "symbol-observable": "^1.2.0",
-    "tar": "^6.2.1",
+    "tar": "^7.4.3",
```

Comment:
I had to update this package to the latest version since v6 did not export the required lower level functions.